docs/index.rst

```diff
@@ -3,8 +3,52 @@ Welcome to "djangocms-versioning"'s documentation!
 
 .. toctree::
    :maxdepth: 3
-   :caption: Contents:
+   :caption: Introduction:
 
    versioning_integration
+
```

Comment:
Doesn't look like you edited `versioning_integration.rst` however I'm getting a build warning when running `make html`:

```
/Users/noeldacosta/Documents/projects/fidelity/djangocms-versioning/docs/versioning_integration.rst:32: WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 7 supplied.

.. code-block:: python
    # blog/models.py
    from django.db import models


    class Post(models.Model):
        pass


    class PostContent(models.Model):
        post = models.ForeignKey(Post, on_delete=models.CASCADE)
        title = models.CharField(max_length=100)
        text = models.TextField()
```
Can this be fixed?